source/deployment/sso-saml-technical.rst

```diff
@@ -0,0 +1,109 @@
+===================================================
+SAML Single-Sign-On (E20): Technical Documentation
+===================================================
+
+Security Assertion Markup Language (SAML) is an open standard that allows identity providers (IdP), like OneLogin, to pass authorization credentials to service providers (SP), like Mattermost.
+
+In simpler terms, it means you can use one set of credentials to log in to many different sites. With OneLogin account, you can log in to Mattermost, Zendesk, Salesforce and other sites securely with the same account.
+
+The main benefit is that it helps administrators centralize user management by controlling which sites users have access to with their OneLogin credentials.
+
+.. contents::
+  :backlinks: top
+  :local:
+
+SAML Providers
+--------------------------------------------
+
+**Identity Providers (IdP)**: An identity provider performs the authentication. When a user clicks to log in, the identity provider confirms who the user is, and sends data to the service provider with the proper authorization to access the site.
+
+Examples: OneLogin, Okta, Microsoft Active Directory (ADFS) or Azure. This is the Wristband Tent.
+
+**Service Providers (SP)**: A service provider gets authentication and authorization information from an IdP such as OneLogin. Once received, it gives the user access to the system and logs the user in.
+
+Examples: Mattermost, Zendesk, Zoom, Salesforce.
+
+SAML Request (AuthNRequest)
+--------------------------------------------
+
+When Mattermost initiates an SP-initiated SAML request flow, it generates a **HTTP-Redirect** binding request to the IdP that contains an XML payload as a base64 string
+
+.. code-block:: none
+
+ bM441nuRIzAjKeMM8RhegMFjZ4L4xPBHhAfHYqgnYDQnSxC++Qn5IocWuzuBGz7JQmT9C57nxjxgbFIatiqUCQN17aYrLn/mWE09C5mJMYlcV68ibEkbR/JKUQ+2u/N+mSD4/C/QvFvuB6BcJaXaz0h7NwGhHROUte6MoGJKMPE=
+
+AuthNRequests can also be signed by Mattermost in v5.14 and later, in which case the XML payload contains the following:
```

Comment:
Contains the following -> similar to

We don’t want people to think this is exactly what it contains, specially the signature and certificate